cloud/endagaweb/templates/dashboard/subscriber_management/subscribers.html

```diff
@@ -0,0 +1,174 @@
+                           value="{{ query }}"/>
+                    <span class="input-group-btn">
+                <button class="btn btn-secondary" type="submit">Search</button>
+              </span>
+
+                </div>
+            </div>
+        </form>
+        {% endif %}
+    </div>
+    <div class='row'>&nbsp;
+        {% for message in messages %}
+        <div class="message alert alert-{{message.tags}}">
+            <a href="#" class="close" data-dismiss="alert">&times;</a>
+            {{message}}
+        </div>
+        {% endfor %}
+    </div>
+    <div class='row'>
+        <div class="col-md-6 left">
+            <div id='number-messages-container'></div>
+        </div>
+    </div>
+
+    <div class='row' id="table_id">
+        <div class="col-xs-12 col-md-12">
+            {% if show_table and number_of_filtered_subscribers %}
+            {% render_table subscriber_table%}
+
+            {% elif show_table and total_number_of_subscribers > 0 %}
+            <h4>No subscribers matched your search.</h4>
+
+            {% elif not show_table and total_number_of_subscribers > 0 %}
+
+            {% else %}
+            <p>
+                There are currently no subscribers registered on this network.
+                Go find some users!
+            </p>
+            {% endif %}
+        </div>
+    </div> <!-- /.row (subscribers) -->
+    <div class='row'>
+        <div class="pull-left" class="container" id="container_id">
+            <table class="table table">
+                <tr>
+                    <label class="radio-inline"><input type="radio"
+                                                       name="subscriberType"
+                                                       id="subscriber"
+                                                       value="Subscriber"><b>Subscriber</b>
+                    </label>
+
+                    <label class="radio-inline"><input type="radio"
+                                                       name="subscriberType"
+                                                       id="retailer"
+                                                       value="Retailer"><b>Retailer</b></label>
+
+                    <label class="radio-inline"><input type="radio"
+                                                       name="subscriberType"
+                                                       id="test_sim"
+                                                       value="Test Sim"><b>Test
+                        SIM</b></label>
+                </tr>
+            </table>
+            <div class='row'>
+
+                <div class="col-xs-6 col-md-4 col-xl-2 pull-left">
+                    <button class='btn btn-primary' id="update" type="Submit">
+                        Update
+                    </button>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+{% else %}
+  {% include 'dashboard/permission_denied.html' %}
```

Comment:
yes we can do this but user will be redirected or kicked out from the view (as no context was there).
will that be right behaviour?